_posts/2016-01-08-smart_video_car.md

```diff
@@ -0,0 +1,48 @@
+# Smart Video Robot Car Kit for Raspberry Pi
+
+This is a short blog about the troubleshooting process we had to face when
+assembling and configuring the Car kit with the Raspberry Pi. We hope it can be useful for everyone, specially for those who are doing it alone and with a limited knowledge of how
+the raspberry pi works.
+
+### Batteries
+
+When we first start reading the manual to get an idea of how to start assembling the Car, we soon realized that the batteries which are actually needed to give the Car the energy it requires to be started were not included in the package at all.
+
+Please be aware of this before purchasing the kit, so make sure you do buy the batteries (x2) required for this kit. It will save you a lot of time and headaches. 
```

Comment:
maybe would be useful to mention that the manual says the batteries required are: x2 18650 of 3.7v rechargeable Li-on battery. We could also mention what we found out about the voltage regulator included in the power board and that we were actually able to use another kind of battery to power it.

The voltage regulartor is a: XL1509 which is a 2A 150KHz 40V Buck DC to DC Converter

so based on that we were able to power the car with a 4 cels 14.8v 1000mAh Li-Po battery (on our own risk) without problems